### DISCUSSION TITLE
UVtool的曝光测试支持Anycubic Photon Mono X吗?如何操作导出切片文件?

### DISCUSSION THREAD

**Discussion by yulei1983** [sn4k3/UVtools#677] [Q&A]
UVtool的曝光测试支持Anycubic Photon Mono X吗?如何操作导出切片文件?

**Comment by sn4k3** [comment]
The normal multi-exposure test does not work with Anycubic. However, you can use the multiple brightness section to archive the "same": https://www.reddit.com/r/AnyCubicPhotonMonoX/comments/11t7gy6/multiple_brightness_test_using_uvtools_mono_x_6k/

PS: Post in English
